bin/sup.sh

```diff
@@ -0,0 +1 @@
+curl -X POST   'https://sentry.io/api/294375/minidump?sentry_key=5428a76c424142128a3ff1c04e5e342e'   -F upload_file_minidump=@$1 -F logs=$2
```

Comment:
@rish9511 Can we do POST request using QT Network libraries instead of using bash script?
If yes, then let's add TODO in the code to do it in future.

Reply:
yes, i'll add this in TODO